fem/bilinearform_ext.hpp

```diff
@@ -70,10 +70,12 @@ class PABilinearFormExtension : public BilinearFormExtension
    const FiniteElementSpace *trial_fes, *test_fes; // Not owned
    /// Attributes of all mesh elements.
    Array<int> elem_attributes, bdr_attributes;
-   mutable Vector tmp_evec; // Work array
+   mutable Vector tmp_evec; // Work arrays
```

Comment:
Why plural, "arrays"?

Reply:
Because in some previous draft of the code there were more than one :)
Fixed in 1bc798642